src/main/resources/public/js/activity.service.js

```diff
@@ -54,45 +54,46 @@
           addTimeForKm(activity, seconds, distanceHalfMarathon);
         }
         // calculate time for every distance starting with nearest km (round ceil)
-        for (var i = Math.ceil(activity.distance); i >= 1; i--) {
+        for (let index = Math.ceil(activity.distance); index >= 1; index -= 1) {
```

Comment:
Sounds good. I will change this in the next hours. For better understanding. All changes have been made according to the rules by eslint. In individual cases I was able to do not use the hard rules, In some other cases we need to discuss.